Formula/d/dwarfs.rb

```diff
@@ -50,6 +50,7 @@ class Dwarfs < Formula
   end
 
   on_linux do
+    depends_on "gcc"
```

Comment:
Might have to do without a Linux bottle since this one has libraries, which, when linked with, will propagate the `gcc` dependency.